ppml/trusted-big-data-ml/python/docker-gramine/base/bigdl-ppml-submit.sh

```diff
@@ -23,11 +23,21 @@ while [[ $# -gt 0 ]]; do
       shift # past argument
       shift # past value
       ;;
+    --driver-memory)
```

Comment:
I remember that the two parameters sgx-executor-memory and sgx-driver-memory were deleted before, why are they added back now?

Reply:
We deleted sgx-memory ones before, as they had been not needed by gramine.
driver-memory and executor-memory are different ones that specify the general memory for spark pods.

There are 3 kinds of memory for spark k8s: general memory (this PR), jvm memory, and sgx memory (deleted).

Reply:
ok I see, thanks for your explanation. Could you also update ppml/README.md to add these two new parameters?

Reply:
sure